commands/management/add_mysql.go

```diff
@@ -21,12 +21,17 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/sirupsen/logrus"
+
+	"gopkg.in/ini.v1"
+
+	"github.com/percona/pmm-admin/agentlocal"
+
```

Comment:
please fix imports to split them  in 3 blocks
1. go libs
2. 3rd party libs
3. local packages

Reply:
here and in other files